tests/pipeline/test_dividends.py

```diff
@@ -204,6 +245,10 @@ def setup(self, dates):
                               ['NaT', '2014-01-04', '2014-01-14'],
                               ['NaT', '2014-01-04']]
         amounts = [['NaN', 1, 15], ['NaN', 7, 13], ['NaN', 3, 1], ['NaN', 23]]
+        currency_types = [[None, "$", "EUR"], [None, "EUR", "$"],
+                          [None, "$", "EUR"], [None, "EUR"]]
```

Comment:
Nope, just 2. It's because in this last case, we don't know anything until the 5th, and then we find out about an announcement in euros, and we don't get any other announcements, so that's what we know until the end of our time frame.